app/javascript/packs/postcodesHelper.js

```diff
@@ -0,0 +1,53 @@
+// Help with finding constituencies from postcodes
+
+$(document).on("click", "#btn-postcode", function() {
```

Comment:
We can use ES6, so consider using arrow functions:

`$(document).on("click", "#btn-postcode", () => {`

Same elsewhere.